jbpm-wb-common/jbpm-wb-data-filters-client/src/main/java/org/jbpm/workbench/df/client/filter/FilterSettingsManager.java

```diff
@@ -40,4 +42,8 @@ void getFilterSettings(String key,
     default FilterSettings getVariablesFilterSettings(List<Long> ids) {
         return null;
     }
+
+    void defaultActiveFilterInit(final Consumer<FilterSettings> existDefaultActiveFilterCallback);
```

Comment:
no need for _final_ keyword in the interface. 
maybe just call it callback?

Reply:
updated